### PR TITLE
Feature: Implement the UI for the Messages and Messaging functionaly

### LINKED ISSUE #8: Feature: Implement Medica web messages
# Description
After a doctor logs in, he or she can be able to view his/ her messages, with the users, he can get the messages related to him, and also be able to reply to them

# Tasks
- [x] Implement the UI as it on figma
- [x] Fetch messages related to the doctor
- [x] Replying to a client
- [x] Making sure to end the session when it is outdated
# Developer Notes
N/A

# Related Issues
N/A

### PR DESCRIPTION
**Pull request by justinmihigo**
#### What does this PR do?
- Implement the UI for the messages where doctors should view the messages sent and be able to send one or multiple
#### Description of Task to be completed?
- [x] Implement the UI as it on figma
- [x] Fetch messages related to the doctor
- [x] Replying to a client
- [x] Making sure to end the session when it is outdated
#### How should this be manually tested?
- Clone the repo, set up the required environment variables, the use npm run dev after you load the link add use the route called messages ``` localhost:3000/messages ```
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker/Trello stories?
- N/A
#### Screenshots (if appropriate)

### CODE REVIEW

**Comment by Adadie** [comment]
@justinmihigo please rebase. I'm also unable to select a chat from the below list
Screenshot 2024-07-09 at 11 45 56

**Comment by Latiah** [comment]
I was able to send a message from the app to the web but there is one thing you could consider the position and the background color of the sender and receiver messages are all the are the same.

Screenshot 2024-07-12 at 12 55 47

**Review by Latiah — APPROVED** [review]

**Comment by fredshema** [comment]
@justinmihigo I managed to send messages on the app and I received them on the web but whenever I send a message using the web It isn't added to messages but can be seen on the app. Also consider loading messages automatically like on the app, right now you have to reload the whole web to get all your messages. Also as suggested by latifah differentiate messages from the doctor and patient all of them look the same. Check the video below for the messages bug:

https://github.com/user-attachments/assets/f0c590b7-fa56-4039-a8dc-56f9ddb28886

File: components/messages/sendMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,315 @@
+// 'use client'
```
Comment (fredshema) [inline components/messages/sendMessage.tsx]: Remove these comments, they are not being used here

File: components/userMessages.ts
Comment (fredshema) [inline components/userMessages.ts:1]: Also remove this file its no longer needed

**Review by fredshema — APPROVED** [review]
It is working well, nice job

**Comment by Adadie** [comment]
When we merge the appointments PR, ensure that doctors are able to start the chat as well

**Pull request merged**